### ISSUE TITLE
Update the links to the E&Js in chronological index

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2617]
**User story**
As an internal stakeholder or outside legal consultant, I need to see the explanation and justification for Commission regulations to do proper legal analysis of them.

**Background:** 
In previous issues, we worked on the citation index. The other main index for the E&Js is the chronological listing of them. (Currently these pages are linked at https://transition.fec.gov/law/cfr/ej_toc.shtml.) This process will work much like the citation index. 

**Work plan**
We need to create the tables in word (done in #2568), update the links, turn into HTML blocks and then populate a page that is set up in Wagtail.

**Completion criteria:**

- [x]  Update the links in those tables to use the better PDFs we uploaded of the older E&Js - Dorothy
(See spreadsheet at https://docs.google.com/spreadsheets/d/1OUELbPCrwJsciY19SkvbxZB14S3MwuayI6Fstob5fiU/edit?usp=sharing for links to change out. Note that with the conversion of SERS to https:// in #2661 , we will need to update the spreadsheet first.)

**Future work:**
Create the resource pages that html blocks will be inserted into - Dorothy #2722 
Use our table to html conversion process to make the html blocks with content team review - Brad or Dorothy/Info specialists #2723

**Comment by dorothyyeager** [comment]
There are some timing issues to work out first for the update of the SERS links to HTTPS. Moving this issue to 8.3 at earliest.

**Comment by dorothyyeager** [comment]
As of now, all E&Js from 1980 on are in the CMS or SERS. I've updated word tables we'll use for the chronological index and I've updated the citation index to capture the newly uploaded E&Js. 

There's some issues with the 1970s E&Js, so most of those are in transition still. The word tables use the transition link in those instances.

With work done, we can close this issue and move on to #2722
